index.d.ts

```diff
@@ -18,9 +18,13 @@ export type Options = {
 	readonly basePath?: string;
 
 	/**
-	Remove any paths that match this regexp from the stack.
+	Remove any paths that return false from this callback.
```

Comment:
We are not just removing the paths, we are removing the stack lines with the matching paths. This should be clearer.